ui/i18n/src/main/res/values/strings.xml

```diff
@@ -61,6 +59,10 @@
     <string name="home_continue_title">Continue listening</string>
     <string name="home_new_title">See what\'s new</string>
     <string name="home_downloads_title">Manage downloads</string>
+    <string name="home_surprise_empty_text">There is just one recent episode, not really much to be surprised here ;)</string>
```

Comment:
This could also be shown if there is no recent episode. I think I would just write something like "no recent unplayed episodes"